docs/examples.rst

```diff
@@ -0,0 +1,9 @@
+.. _examples:
+
+Examples of using skops
```

Comment:
I think the "of using skops" part is a bit redundant inside of the skops documentation ;)

Reply:
```suggestion
Examples of interactions with the Hugging Face Hub
```